docs/reference/sql/functions/math.asciidoc

```diff
@@ -8,68 +8,275 @@ to be numeric.
 
 ==== Generic
 
-* `ABS`
+[[sql-functions-math-abs]]
+===== `ABS`
 
-https://en.wikipedia.org/wiki/Absolute_value[Absolute value], returns \[same type as input]
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+ABS(numeric_exp<1>)
```

Comment:
One of the things that I found when we converted the rest of the ES docs to `// CONSOLE` to make them automatically tested was that as a side effect of the change I converted a lot of examples from "unrooted" like this one to "complete". So, like, this one would end up as `SELECT ABS(score <1>) FROM game` or something. At the time I thought that it was sort of useless ceremonial words around the real example. A few months later I had someone interrupt a conversation I was having to tell me that "completing" all of those examples really helped them understand Elasticsearch better. So, that was a long way of saying, I think this should be a full `SELECT` statement and it may as well come from the `docs.csv-spec` file in that case.

Reply:
Hm interesting. For some of these functions would be difficult to come up with something in terms of actual test data, especially the trigonometric functions where I tried to "demonstrate" a small formula combining two or more functions.
If it's only about the synopsis itself, then yeah, I can add a more "real-life" SELECT.

Reply:
I think something like
```
SELECT ABS(-10) AS result;

    result     
---------------
10    
```

might be nice too.

Reply:
But those do exist. They are in `docs.csv-spec` and for each function they are referenced as part of `include-tagged::{sql-specs}/docs.csv-spec[abs]` where `abs` comes from 

```
mathInlineAbs
// tag::abs
SELECT ABS(-123.5), ABS(0.1), ABS(55);
   ABS(-123.5)  |   ABS(0.1)    |    ABS(55)
---------------+---------------+---------------
123.5          |0.1            |55
// end::abs
;
```